src/components/activity-header/account-owner.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import AccountOwnerIcon from "../../assets/svg-icons/account-circle-icon.svg";
+
+import "./account-owner.scss";
+
+interface IProps {
+  userName: string;
+}
+export class AccountOwnerDiv extends React.PureComponent <IProps> {
```

Comment:
Not a big deal, but we tend to not use div and span in our component names (it might make it harder to refactor or imply certain layout patterns that might not hold true).  I would probably just call this AccountOwner or AccountOwnerContainer if that feels too short.